nl.sarine.gpxViewer.json

```diff
@@ -0,0 +1,166 @@
+{
+    "app-id" : "nl.sarine.gpxViewer",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "master",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "gpx-viewer",
+    "tags" : [
+        "devel"
+    ],
+    "desktop-file-name-prefix" : "(Development) ",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=session-bus",
```

Comment:
Why does it need full access to the session bus?

Reply:
without it program crashes, but didn't investigate why it needs that access

```
Aplicación iniciada a las 12:11:55
                                  Failed to register: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown
Aplicación terminada
```

Reply:
You need to run the app with --log-session-bus to identify the names it needs to call and then allowlist them specifically. This permission is too broad to add without specific justification.

Reply:
it uses GtkApplication to keep one unique instance, instead of launching multiple.

Reply:
I believe this isn't resolved, since the permission is still there.

Preferably, the app should register itself on dbus under the same ID as the flatpak ID. But if this isn't possible, you can add `--own-name=` permission with the ID the app registers itself as.